lib/go/test/fuzz/fuzz.go

```diff
@@ -24,10 +24,10 @@ package fuzz
 import (
 	"context"
 	"fmt"
-	"shared"
 	"strconv"
-	"tutorial"
 
+	"github.com/apache/thrift/lib/go/test/fuzz/gen-go/shared"
+	"github.com/apache/thrift/lib/go/test/fuzz/gen-go/tutorial"
```

Comment:
cc @catenacyber : I changed the import paths of these 2 so that we do not need to do `go mod edit --replace` inside thrift's CI. I believe in oss-fuzz you can still do `go mod edit --replace` to replace these 2 to something local, but please give that a try and let me know if that doesn't work.

Reply:
Thanks @fishy 
Ok, let me know when you merge this go module, and I will go back to adapt oss-fuzz PR

Reply:
can you try it from my branch instead? I'd rather not to merge it if it doesn't work :)

Reply:
Well, I guess this branch is about supporting go modules for all of thrift.
When this will me merged, I do not doubt I can manage to adapt the oss-fuzz PR

If you want, I can test it with this branch, but that may take a few days

Reply:
OK that makes sense. I'll merge this soon and if it turns out to break oss-fuzz, we can make follow up changes to fix them.

Reply:
You were right, I do not get it to work with this change
As the package is named `shared` elsewhere and not `github.com/apache/thrift/lib/go/test/fuzz/gen-go/shared`
Could you revert this ?
Sorry for missing it in the first place

Reply:
@catenacyber https://github.com/apache/thrift/pull/2358